practice_app/pages/index.js

```diff
@@ -80,19 +77,19 @@ export default function Home({ context }) {
         }
     }
 
-    const getNumber = async () => {
-        if (countr == null) {
-            setNumb('Please select a location by clicking on the map')
+    const getCovidStats = async () => {
+        if (country.name == null) {
```

Comment:
This will crash if `country` itself is undefined

Reply:
But country is already defined globally in this lines: const [country, setCountry] = useState({
        code: 'TR',
        name: 'Turkey',
    })

Reply:
You are right. I had in mind some earlier code where `setCountry(undefined)` could happen.